scene/resources/2d/tile_set.h

```diff
@@ -940,8 +943,17 @@ class TileData : public Object {
 	void set_y_sort_origin(int p_y_sort_origin);
 	int get_y_sort_origin() const;
 
+#ifndef DISABLE_DEPRECATED
 	void set_occluder(int p_layer_id, Ref<OccluderPolygon2D> p_occluder_polygon);
 	Ref<OccluderPolygon2D> get_occluder(int p_layer_id, bool p_flip_h = false, bool p_flip_v = false, bool p_transpose = false) const;
+#endif // DISABLE_DEPRECATED
+
+	void set_occluder_polygons_count(int p_layer_id, int p_polygons_count);
+	int get_occluder_polygons_count(int p_layer_id) const;
```

Comment:
```suggestion
	void set_occluder_polygon_count(int p_layer_id, int p_polygons_count);
	int get_occluder_polygon_count(int p_layer_id) const;
```
English doesn't use plurals for nouns in this case